app/javascript/mastodon/locales/zh-CN.json

```diff
@@ -5,27 +5,27 @@
   "account.edit_profile": "修改个人资料",
   "account.follow": "关注",
   "account.followers": "关注者",
-  "account.follows": "正关注",
```

Comment:
I was doing three characters for some local alignment. It's fun to have some sort of rhythms…